zone/exp.cpp

```diff
@@ -1229,7 +1229,7 @@ void Raid::SplitExp(ExpSource exp_source, const uint64 exp, Mob* other) {
 	const auto highest_level   = GetHighestLevel();
 
 	if (RuleB(Character, EnableRaidEXPModifier)) {
-		raid_experience = static_cast<uint64>(static_cast<float>(raid_experience) *	(1.0f - RuleR(Character, RaidExpMultiplier)));
+				raid_experience = static_cast<uint64>(static_cast<float>(raid_experience) *	(1.0f - RuleR(Character, RaidExpMultiplier)));
```

Comment:
Why did this get touched ?

Reply:
Oh weird, didn't notice that, must have had the file open on accident.